.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+(description of the change)
+
+#### Pull Request Checklist ##
+- [ ] Passed code style checking (`./pre-commit.sh` or manual run of pylint & mypy)
+- [ ] Changes are complete (i.e. I finished coding on this PR)
```

Comment:
Quick suggestion: add a note for work-in-progress pull requests for collaboration/feedback.  It could be something like "If posting work-in-progress for feedback, prefix your pull request title with [WIP] until you can check this box."